### ISSUE TITLE
Make "Save" and "Approve" the same

### ISSUE THREAD

**Issue by maximejobin** [GlotPress/GlotPress-WP#1014]
I'm a LTE. I spend most of my time accepting and editing translations made by others.

If I see a string and want it as is, I must "Accept".
If I edit a string, I must "Save".

Otherwise, it displays an error message.

I think that no matter if I click on "Accept" or "Save", the wanted behavior should be automatically detected.

This is simple, not confusing and would save me (and probably others) a lot of time.

**Comment by Mte90** [comment]
I don't think that should be the same, because you want a review from someone else as example.
Everyone need a review because no one can avoid errors and if we push this we create a not balanced capabilities system, like with gatekeepers and also suggest a toxic behaviour on long term.
If we force this behaviour for all of them means like that a LTE/GTE has more power then others and doesn't need to follow the rules because no one can watch them.

**Comment by maximejobin** [comment]
@Mte90 I'm sorry but I don't understand what you mean. LTE's and GTE's translations are _not_ reviewed.

As a GTE or LTE, whatever I propose is automatically marked as "current". Therefore, if I translate something, it is already marked as "Suggested + Accepted" when I click on "Save". If a non-GTE suggests a string, clicking on "Save" gives an error. I'm not claiming I should be marked as the translator of the string I approved by clicking "Save".

For the explanation above, I don't understand what would be toxic. As a GTE or LTE, we already have more power.

>Everyone need a review because no one can avoid errors...

I'm not against that claim but that is not how it works in my locale (fr-CA).

Is there something I don't get?

**Comment by Mte90** [comment]
If you are talking about GTE/LTE is not something for GlotPress because this roles doesn't exists in this plugin.
Translate.wp.org implement this capabilities and their features, this project is only for the plugin that is one of the pieces of the wp platform.
I suggest to you to open a ticket on https://meta.trac.wordpress.org :-)

We cannot implement this in glotpress because this roles doesn't exist and also because I already explained above the reasons.

**Comment by toolstack** [comment]
@maximejobin you should probably open this up as an issue on the meta trac over a wordpress.org.

**Comment by maximejobin** [comment]
Funny, I opened it on meta trac and it was suggested to report it here and on GlotDict. So did I. Then, you're telling me to open it on meta!

I'm confused...

https://meta.trac.wordpress.org/ticket/4496

**Comment by Mte90** [comment]
Replied on the trac ticket

**Comment by ocean90** [comment]
Just replace LTE with validator and you have the same in a default GlotPress install.

Related: https://github.com/GlotPress/GlotPress-WP/commit/39e768afeba5c60d2f22c84ff413dd2cf129a31a / #707
